.github/workflows/ci-experimental.yml

```diff
@@ -116,3 +116,11 @@ jobs:
           fi
 
           make V=1 LOG_LEVEL=DEBUG QUICK_AND_DIRTY_COMPILER=1 test testwakunode2
+
+  build-docker-image:
+    needs: changes
+    if: ${{ needs.changes.outputs.v2 == 'true' || needs.changes.outputs.common == 'true' || needs.changes.outputs.docker == 'true' }}
```

Comment:
Is is needed to add the next snippet in the "filter change detection" above?
```
          docker:
          - 'docker/**'
```

Reply:
Ah, yes, I added that for the ci, but not for the ci-experimental, good catch